test cases/common/224 test priorities/testprog.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
+
+import sys
+
+print('"{}"'.format(sys.argv[1]))
```

Comment:
This will not print the same thing as the above shell script. I think you just want `print(sys.argv[1])`.